ozaria/site/views/play/level/tome/SpellView.coffee

```diff
@@ -960,11 +966,7 @@ module.exports = class SpellView extends CocoView
     beginningOfLine = not currentLine.substr(0, cursorPosition.column).trim().length  # uncommenting code, for example
     incompleteThis = /^(s|se|sel|self|t|th|thi|this|g|ga|gam|game|h|he|her|hero)$/.test currentLine.trim()
     #console.log "finished=#{valid and (endOfLine or beginningOfLine) and not incompleteThis}", valid, endOfLine, beginningOfLine, incompleteThis, cursorPosition, currentLine.length, aether, new Date() - 0, currentLine
-    if not incompleteThis and @options.level.isType('game-dev')
-      # TODO: Improve gamedev autocast speed
-      @spell.transpile @getSource()
-      @cast(false, false, true)
-    else if (endOfLine or beginningOfLine) and not incompleteThis
+    if (endOfLine or beginningOfLine) and not incompleteThis
```

Comment:
It seems it will now run for game-dev levels also, is that intended?

Reply:
Preloading is OK, but automatically casting is not. I believe that without this, neither the if nor the else gets called for game dev levels, which can block the full loading of the level. 

Now that we want to be selective about updating the level from writing code, it should behave quite similarly for game dev and non game dev levels.